care/emr/signals/charge_item_def/product.py

```
@@ -0,0 +1,43 @@
+        old_instance = Product.objects.get(id=instance.id)
+        setattr(
+            instance, _OLD_CHARGE_ITEM_DEF_ID, old_instance.charge_item_definition_id
+        )
+        setattr(instance, _OLD_DELETED, old_instance.deleted)
+    else:
+        setattr(instance, _OLD_CHARGE_ITEM_DEF_ID, None)
+        setattr(instance, _OLD_DELETED, False)
+
+
+@receiver(post_save, sender=Product)
+def update_separately_billable_product(sender, instance, created, **kwargs):
```

Ruff (0.14.13)

Unused function argument: `sender`

(ARG001)